### PR TITLE
update doc for rapids v2202 release

### PR DESCRIPTION
**Pull request by nvliyuan**
Signed-off-by: liyuan <yuali@nvidia.com>
update doc for rapids v2202 release

### CODE REVIEW

**Comment by nvliyuan** [comment]
we should add the remove existing xgboost4j jars step in init_script for Databricks 7.3 LTS ML env:
`-r-xr-xr-x 1 root root 282315103 Feb 14 09:18 spark--maven-trees--ml--7.x--xgboost--ml.dmlc--xgboost4j_2.12--ml.dmlc__xgboost4j_2.12__1.0.0.jar`
`-r-xr-xr-x 1 root root    496561 Feb 14 09:19 spark--maven-trees--ml--7.x--xgboost--ml.dmlc--xgboost4j-spark_2.12--ml.dmlc__xgboost4j-spark_2.12__1.0.0.jar`
and for Databricks 9.1 LTS ML env:
`-r-xr-xr-x 1 root root  270M Feb 14 08:56 spark--maven-trees--ml--9.x--xgboost-gpu--ml.dmlc--xgboost4j-gpu_2.12--ml.dmlc__xgboost4j-gpu_2.12__1.4.1.jar`
`-r-xr-xr-x 1 root root  485K Feb 14 08:56 spark--maven-trees--ml--9.x--xgboost-gpu--ml.dmlc--xgboost4j-spark-gpu_2.12--ml.dmlc__xgboost4j-spark-gpu_2.12__1.4.1.jar`

**Comment by nvliyuan** [comment]
not update PCA demo version because it would be updated in this #99

**Review by viadea — DISMISSED** [review]

File: docs/get-started/xgboost-examples/csp/databricks/generate-init-script-7.3.ipynb [hunk 1]
```diff
@@ -130,6 +130,23 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rm -f /databricks/jars/spark--maven-trees--ml--7.x--xgboost--ml.dmlc--xgboost4j_2.12--ml.dmlc__xgboost4j_2.12__1.0.0.jar\n",
+    "rm -f /databricks/jars/spark--maven-trees--ml--7.x--xgboost--ml.dmlc--xgboost4j-spark_2.12--ml.dmlc__xgboost4j-spark_2.12__1.0.0.jar"
+   ]
```
Comment (GaryShen2008) [inline docs/get-started/xgboost-examples/csp/databricks/generate-init-script-7.3.ipynb:148]: I think we overwrite these 2 files in the above cell.
sudo cp /dbfs/FileStore/jars/xgboost4j_3.0-1.4.2-0.2.0.jar /databricks/jars/spark--maven-trees--ml--7.x--xgboost--ml.dmlc--xgboost4j_2.12--ml.dmlc__xgboost4j_2.12__1.0.0.jar\n
sudo cp /dbfs/FileStore/jars/xgboost4j-spark_3.0-1.4.2-0.2.0.jar /databricks/jars/spark--maven-trees--ml--7.x--xgboost--ml.dmlc--xgboost4j-spark_2.12--ml.dmlc__xgboost4j-spark_2.12__1.0.0.jar\n
Why do we need to remove it again?

File: examples/Spark-cuML/pca/spark-submit.sh [hunk 2]
```diff
@@ -39,4 +39,4 @@ $SPARK_HOME/bin/spark-submit \
 --conf spark.network.timeout=1000s \
 --jars $ML_JAR,$CUDF_JAR,$PLUGIN_JAR \
 --class com.nvidia.spark.examples.pca.Main \
-/workspace/target/PCAExample-21.12.0-SNAPSHOT.jar
+/workspace/target/PCAExample-22.02.0-SNAPSHOT.jar
```
Comment (GaryShen2008) [inline examples/Spark-cuML/pca/spark-submit.sh:42]: Will we change the PCAExample's version to match our release version?

File: docs/get-started/xgboost-examples/csp/databricks/generate-init-script.ipynb [hunk 3]
```diff
@@ -123,6 +123,23 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rm -f /databricks/jars/spark--maven-trees--ml--9.x--xgboost-gpu--ml.dmlc--xgboost4j-spark-gpu_2.12--ml.dmlc__xgboost4j-spark-gpu_2.12__1.4.1.jar\n",
+    "rm -f /databricks/jars/spark--maven-trees--ml--9.x--xgboost-gpu--ml.dmlc--xgboost4j-gpu_2.12--ml.dmlc__xgboost4j-gpu_2.12__1.4.1.jar"
```
Comment (GaryShen2008) [inline docs/get-started/xgboost-examples/csp/databricks/generate-init-script.ipynb:140]: Same comment as above? Why we need to delete?

File: docs/get-started/xgboost-examples/csp/databricks/generate-init-script-7.3.ipynb (hunk 1 above)
Comment (nvliyuan) [reply docs/get-started/xgboost-examples/csp/databricks/generate-init-script-7.3.ipynb:148]: yes, we don't need to remove the jars, the init script works, verified as below:
image

File: examples/Spark-cuML/pca/spark-submit.sh (hunk 2 above)
Comment (nvliyuan) [reply examples/Spark-cuML/pca/spark-submit.sh:42]: in Spark-cuML/pca/pom.xml, the version is 22.02.0-SNAPSHOT, so I think this script should also keep the same, @wjxiz1992 could you please help to check the version?

File: docs/get-started/xgboost-examples/csp/databricks/generate-init-script.ipynb (hunk 3 above)
Comment (nvliyuan) [reply docs/get-started/xgboost-examples/csp/databricks/generate-init-script.ipynb]: reverted

**Review by GaryShen2008 — APPROVED** [review]
Let's wait for v22.02 release.

**Comment by nvliyuan** [comment]
> Let's wait for v22.02 release.

yes, I will create a main branch after release

**Comment by nvliyuan** [comment]
squash and merge this pr since rapids plugin v22.02.0 is released

**Pull request merged**